.gitignore

```diff
@@ -0,0 +1,3 @@
+/output/
+vendor/
```

Comment:
I didn't notice the `vendor` dir in the root, maybe because of a different configuration of the composer! so it is accepted

Reply:
`/dir/` is any dir that start at the root of git
`vendor/` is any dir with name vendor in all the repo, so any `framework-xx/vendor/` and /**